### ISSUE TITLE
MUC menu - bookmark string not changed nor icon

### ISSUE THREAD

**Issue by licaon-kter** [conversejs/converse.js#2236]
HEAD (4dedfdb89ea8a342d78186ea2bb7dc63ec19aa46), Firefox 80, using IndexDB

**To Reproduce**
1. Make a private MUC in Conversations (for OMEMO)
2. invite Converse user
3. Accept and post
4. Press menu-> Bookmark
5. log out, log in
6. open muc
7. press menu

**Expected behavior**
Have Unbookmark
Press Unbookmark
Go back to menu see entry still reads "Unbookmark"
See that bookmark icon in room title is still present
Press Unbookmark again, see that you are actually asked to bookmark it

Basically the menu entry (and icon) don't register that the muc status (bookmarked or not) has changed.

Also, at step 5, going to menu after the initial bookmark action I could see that the entry has not changed to _Unbookmark_

**Comment by licaon-kter** [comment]
Appears fixed now
